python/social_bot/envs/play_ground.py

```diff
@@ -91,6 +93,7 @@ def __init__(self,
             with_language (bool): The observation will be a dict with an extra sentence
             use_image_observation (bool): Use image, or use low-dimentional states as
                 observation. Poses in the states observation are in world coordinate
+            max_steps (int): episode will end in so many steps, will be passed to the tasks
```

Comment:
episode will end "with negative reward" in so many steps "if goal is not reached"?

Reply:
That's true for goal task, but for other tasks there might not be negative reward.